scripts/benchmarks/configuration.sh

```diff
@@ -6,15 +6,15 @@ EXTERNAL_WEIGHTS_PATH="./runtime/common/src/weights/"
 export FRAME_PALLETS=( frame_system pallet_balances pallet_bounties pallet_democracy \
                 pallet_identity pallet_membership  pallet_multisig pallet_preimage \
                 pallet_proxy pallet_scheduler pallet_timestamp pallet_treasury \
-                pallet_utility pallet_vesting \
-                ) # pallet_collective pallet_grandpa )
+                pallet_utility pallet_vesting pallet_collective ) # pallet_grandpa )
```

Comment:
why pallet_collective was earlier disabled?

Reply:
The benchmarks for `pallet_collective` failed before, now they work. See https://github.com/zeitgeistpm/zeitgeist/issues/822.

Reply:
okay but there is still some investigation required for #822 or it failure was due to parity's bug and it got fixed?

Reply:
It's failure was due to a bug in `pallet_collective` (uint bitwidth mismatch) and I am not sure if it got fixed, but we use a configuration that does not trigger that bug currently (we don't suffer any drawbacks from that either).